bundles/specmate-migration/src/com/specmate/migration/basemigrators/AttributeAddedBaseMigrator.java

```diff
@@ -0,0 +1,109 @@
+package com.specmate.migration.basemigrators;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import com.specmate.common.SpecmateException;
+
+public abstract class AttributeAddedBaseMigrator {
+	protected Connection connection;
+	
```

Comment:
A more outspoken name for the methods would be migrateNewStringAttribute

Reply:
Yes, makes sense.